sos/report/plugins/kernel.py

```diff
@@ -107,6 +107,10 @@ def setup(self):
             "/var/log/dmesg",
             "/sys/fs/pstore",
             "/sys/kernel/debug/dynamic_debug/control",
+            "/proc/pagetypeinfo",
+            "/proc/buddyinfo",
```

Comment:
We already collect it 20 lines above :)

Reply:
yes.. we already have this info.. will remove this info

Reply:
will update and send new version of patch..Thanks